<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>41</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>280</width>
    <height>40</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>280</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Start Thread 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>Stop Thread 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
